DirectXMesh/DirectXMeshP.h

```diff
@@ -9,6 +9,7 @@
 // http://go.microsoft.com/fwlink/?LinkID=324981
 //-------------------------------------------------------------------------------------
 
+#ifdef _MSC_VER
 #pragma once
```

Comment:
While pragma once is not an official part of the standard, it should be a DeFacto standard at this point. What compiler complained about it?